hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/BootstrapExecutor.java

```diff
@@ -170,10 +170,11 @@ private void initializeTable() throws IOException {
       throw new HoodieException("target base path already exists at " + cfg.targetBasePath
           + ". Cannot bootstrap data on top of an existing table");
     }
-
+    SparkHoodieIndex index = SparkHoodieIndex.createIndex(bootstrapConfig);
```

Comment:
same comment. can we avoid creating the index object here?

Reply:
done